src/Pim/Bundle/CatalogBundle/spec/Doctrine/Common/Saver/FamilySaverSpec.php

```diff
@@ -51,8 +51,8 @@ function it_saves_multiple_family(
         $objectManager->persist($family1)->shouldBeCalled();
         $objectManager->persist($family2)->shouldBeCalled();
 
-        $completenessManager->scheduleForFamily($family1)->shouldBeCalled($family1);
-        $completenessManager->scheduleForFamily($family2)->shouldBeCalled($family2);
+//        $completenessManager->scheduleForFamily($family1)->shouldBeCalled($family1);
```

Comment:
I already added it in the FamilySaver. The phpspec will have to be updated at the same time.